src/pages/fundamentals/pages/appendix/pages/config-annotations/pages/view/pages/menuPanel.adoc

```diff
@@ -27,20 +32,30 @@ public static class VMPView {
     private String showView;
 
     @Label("Appearance")
-    @MenuPanel
+    @MenuPanel(flow="domainAalias")
     private VMPAppearance appearance;
 
     @Model
     @Getter @Setter
     public static class VMPAppearance {
 
         @Label("Toggle Full Screen")
-        @MenuLink
+        @MenuLink(url="domainAalias/page1")
         private String toggleFullScreen;
 
         @Label("Toggle Always on Top")
-        @MenuLink
+        @MenuLink(url="domainBalias/page1")
         private String toggleAlwaysOnTop;
+
+        @Label("Toggle with Animation")
+        @MenuLink(url="domainAalias/page2")
+        private String toggleWithAnimation;
     }
 }
 ----
+
+
+In the above configuration, the menulinks have a url that points to either domainAalias or domainBalias. Lets say VMPView belongs to the domain that is the layout of the domainA. In other words, the view domain A has the layout of just a menupanel and menulink. So initially, When 'toggle full screen' menulink is clicked the breadcrumb would say - Home->Toggle Full Screen
```

Comment:
1. surround code related terms like domainAalias with backticks
2. Consider using [callouts](https://asciidoctor.org/docs/user-manual/#callouts).
3. rename components to use backticks and proper capitilization `MenuPanel`, `MenuLink`